packages/sq/src/Select.ts

```diff
@@ -2,20 +2,25 @@ import Builder from "./Builder";
 
 type Pluck = string | [string, string];
 
+export type SelectLoad = {
+  keys: [string, string];
+  select: Select;
+};
+
+export type SelectJoin = {
+  keys: [string, string];
+  tableName: string;
+  select: Select;
```

Comment:
I think at some point we should make the context getter flatten all of the contexts, so that you never have to call `.context` anywhere in the adapter.